app/views/spree/products/show.html.erb

```diff
@@ -1,51 +1,359 @@
-
-  <%= products_structured_data([@product]) %>
+  <div id="related-products"></div> -->
 
-  <%= render 'spree/shared/product_added_modal' %>
-<% end %>
+  <%#= products_structured_data([@product]) %>
 
+  <%#= render 'spree/shared/product_added_modal' %>
+<%# end %>
+<% @body_id = 'product-details' %>
 <div
   data-related-products
   data-related-products-id="<%= @product.slug %>"
   data-related-products-enabled="<%= @product.respond_to?(:has_related_products?) %>" />
+  <main class="page-contents">
+    <div class="row no-gutters">
+      <div class="col-xl-1 tablet-hide"></div>
+      <div class="col-xl-8 col-lg-8">
+        <div class="product-detail-wrapper">
+            <%= spree_breadcrumbs(@taxon, '', @product) %>
+          <div class="title-wrap position-relative">
+            <h1 class="title"> <%= @product.name %></h1>
+            <div class="ratings">  <%= image_tag "star.svg", alt:'star' %> <%= image_tag "star.svg", alt:'star' %> <%= image_tag "star.svg", alt:'star' %> <%= image_tag "star.svg", alt:'star' %> <%= image_tag "star-off.svg", alt:'star' %><span>5 Reviews</span></div>
+            <ul class="product-standards d-flex flex-wrap align-items-center phone-visible">
+              <li class="glutenfree-icon" data-toggle="tooltip" data-placement="bottom" title="Gluten free">Gluten free</li>
+              <li class="nongmo-icon" data-toggle="tooltip" data-placement="bottom" title="Non GMO">Non GMO</li>
+              <li class="u-icon" data-toggle="tooltip" data-placement="bottom" title="U">u</li>
+              <li class="vegan-icon" data-toggle="tooltip" data-placement="bottom" title="Vegan">Vegan</li>
+            </ul>
+          </div>
+          <div class="product-desc-wrap" data-hook="product_right_part"> 
+            <div class="row" data-hook="product_show">
+              <%= render partial: 'gallery_modal' %>
+              <div class="col-xl-5">
+                <%= render partial: 'phone_cart' %>
+                <div  id="product-description" data-hook="product_right_part_wrap">
+                  <%= render partial: 'description' %>
+                </div>
+               </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <%= render 'cart_form', variant_change_identifier: 'productCarousel' %>   
+      <div class="product-tabs-wrapper">
+        <div class="row no-gutters">
+          <div class="col-xl-1 tablet-hide"></div>
+          <div class="col-xl-7 col-lg-7" data-hook="product_properties">
+          <%= render partial: 'properties' %>
+          </div>
+          <%= render partial: 'related_products' %>
+        </div>
+        <%= render partial: 'customer_reviews' %> 
+      </div>
+    </div>
+  </main>
+</div>
+
+
+<script>
+  AOS.init();
+</script> 
+
+<!-- For mobile menu --> 
+<script>
+  const slinky = $('#menu').slinky({title:true});
+</script> 
+<script>
+  $(function () {
+    $('[data-toggle="tooltip"]').tooltip()
+  })
```

Comment:
create new file js

Reply:
Create new js file under assets with name: 'product_details.js' and move all the js there.